sql/catalyst/src/main/scala/org/apache/spark/sql/connector/catalog/CatalogV2Util.scala

```diff
@@ -220,4 +220,8 @@ private[sql] object CatalogV2Util {
       case _: NoSuchDatabaseException => None
       case _: NoSuchNamespaceException => None
     }
+
+  def isSessionCatalog(catalog: CatalogPlugin): Boolean = {
```

Comment:
Will this need to be renamed later?

Reply:
Yea, we will update `CatalogManager.SESSION_CATALOG_NAME` later and also update this one to `isSparkCatalog`